src/utils/pipes/parse.optional.enum.pipe.ts

```diff
@@ -6,7 +6,7 @@ export class ParseOptionalEnumPipe<T extends { [name: string]: any }> implements
   transform(value: string | undefined, _: ArgumentMetadata): Promise<string | undefined> {
     return new Promise(resolve => {
       if (value === undefined || value === '') {
-        return resolve(undefined);
+        return resolve('');
```

Comment:
Same as the above comment. There are service-level checks for undefined as an explicit value